src/main/java/hudson/plugins/promoted_builds/conditions/ManualCondition.java

```diff
@@ -52,8 +55,31 @@
  */
 public class ManualCondition extends PromotionCondition {
     private String users;
+        
     private List<ParameterDefinition> parameterDefinitions = new ArrayList<ParameterDefinition>();
     
+    
+    //Code to fetch Approve Promotion Button label from properties file
+    private String executeButtonLabel;
+    
+    public String getExecuteButtonLabel() {
```

Comment:
Add Javadoc. Also should include <code>@since TODO</code> and a meaningful description.